app/src/test/java/mobile/programming/musicApp/InstrumentDataIntegrationTest.kt

```diff
@@ -32,10 +32,10 @@ class InstrumentDataIntegrationTest {
     }
 
     private fun checkIfInstrumentDataIsEmpty() {
-        var instrumentlist = ArrayList<Instrument>()
+        var instrumentList = ArrayList<Instrument>()
         mvm.instruments.observeForever {
 
-            instrumentlist = it
+            instrumentList = it
```

Comment:
This is just another instance of the instrumentList variable that needs to be changed to lower camelcase.